examples/transmitter/swagger_server/events.py

```diff
@@ -29,18 +29,49 @@ class CAEPEvent(Event):
     initiating_entity: Optional[str]
     reason_admin: Optional[Dict[str, str]]
     reason_user: Optional[Dict[str, str]]
+    __uri__ = "https://schemas.openid.net/secevent/caep/event-type/"
 
 
 class SessionRevoked(CAEPEvent):
-    __uri__ = "https://schemas.openid.net/secevent/caep/event-type/session-revoked"
+    pass
+    
+    
+class TokenClaimsChange(CAEPEvent):
+    claims: Dict[str, str]
+
+
+class CredentialChange(CAEPEvent):
+    credential_type: str
```

Comment:
`credential_type` and `change_type` should be Enums because they can only take on specific values